Remotion/Web/Core/Res/Themes/NovaViso/HTML/TabbedMenu.css

```diff
@@ -73,31 +73,36 @@ div.tabbedMainMenu span.tabStripTabSelected a,
 div.tabbedSubMenu span.tabStripTab a,
 div.tabbedSubMenu span.tabStripTabSelected a
 {
-  border: none;
+  border: unset;
+  background-color: unset;
```

Comment:
After seeing this in action, `unset` just feels more explicit and consistent than setting `transparent`